weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/StructuredFeedback/HumanAnnotation.tsx

```diff
@@ -194,12 +194,23 @@ const FeedbackComponentSelector: React.FC<{
     );
 
     switch (type) {
+      case 'integer':
+        return (
+          <NumericalFeedbackColumn
+            min={jsonSchema.minimum}
```

Comment:
make sure your are respecting https://json-schema.org/understanding-json-schema/reference/numeric#range correctly (inclusive vs non inclusive)

Reply:
Yeah it should be inclusive by default, which is what the component expects. Rejecting iff: `(min != null && parsedVal < min) || (max != null && parsedVal > max)` so accepting when `val == min`